pkg/rpc/convert.go

```diff
@@ -892,9 +892,7 @@ func ConvertToRPCPutBlobRequest(diffID string, blobInfo ftypes.BlobInfo) *cache.
 	var customResources []*common.CustomResource
 	for _, res := range blobInfo.CustomResources {
 		data, err := structpb.NewValue(res.Data)
-		if err != nil {
-
-		} else {
```

Comment:
it's not related to the linter
But it looks like there's a bug here and there should be a warning, like in other cases. For example:
https://github.com/aquasecurity/trivy/blob/40f4c065603b41867ef08269b2d8d8ef6c7ea38f/pkg/rpc/convert.go#L111-L113

maybe we can fix this right away?
cc. @knqyf263

Reply:
I also noticed and accepted the change as the function doesn't return an error. But yes, you're right. I forgot about the idea of showing warnings. We should warn it.